website/docs/r/cloudwatch_event_target.html.markdown

```diff
@@ -68,11 +68,11 @@ The following arguments are supported:
 
 * `rule` - (Required) The name of the rule you want to add targets to.
 * `target_id` - (Optional) The unique target assignment ID.  If missing, will generate a random, unique id.
-* `arn` - (Required) The Amazon Resource Name (ARN) associated of the target.
+* `arn` - (Required) The Amazon Resource Name (ARN) associated of the target. (It should be the ID of the ECS Cluster if `ecs_target` is used.)
```

Comment:
As you exposed it in your issue, the `aws_ecs_cluster.*.id` attribute is the ARN. Since the related documentation exposes it, we should not redo the work here, especially since I'm agrai it will confuse people :/
The other reason is that there are so many edge cases in so many resources, that adding such indications would result in enormous documentation, which we will try to avoid 😄 

However, one thing that could be done is the normalization of the data source & resource, so that both uses the `arn` too. Does it seem reasonnable to you?